market/mk20/http/http.go

```diff
@@ -130,7 +130,8 @@ func APIRouter(mdh *MK20DealHandler, domainName string) http.Handler {
 	SwaggerInfo.Version = version
 	mux := chi.NewRouter()
 	mux.Use(dealRateLimitMiddleware())
-	mux.Use(AuthMiddleware(mdh.db))
+	// Disabled, far too complex to deal with for now
+	// mux.Use(AuthMiddleware(mdh.db))
```

Comment:
Couldn't get this to work, too complex particularly when working with Ethereum addresses and this is built for Filecoin addresses using the specific Go encoding forms that aren't as readily accessible in other languages. This auth thing needs a rethink to be more accessible and usable.